helm_deploy/README.md

```diff
@@ -11,7 +11,7 @@ version.BuildInfo{Version:"v3.0.1", GitCommit:"7c22ef9ce89e0ebeb7125ba2ebf7d421f
 
 - Ensure a TLS cert for your intended hostname is configured and ready, see section below.
 
-### Useful helm (v3) commands:
```

Comment:
Was a [NBSP](https://en.wikipedia.org/wiki/Non-breaking_space), which meant the title wasn't displaying as expected

Reply:
Think this might have come from the template. I'll check that as well